include/flatbuffers/flexbuffers.h

```diff
@@ -1181,6 +1181,7 @@ class Builder FLATBUFFERS_FINAL_CLASS {
   }
 
   // Overloaded Add that tries to call the correct function above.
+  void Add() { Null(); }
```

Comment:
The whole point of Add() is that it is overloaded, so that it can be used in contexts where the type of the argument is not know (e.g. a template). The version you added is not overloaded (by type) so has no such use. In that case, using `Null` is better as it is much more informative.

Reply:
We should rename the api Add taking multiple arguments then, to something AddPair or something else.